lib/elixir/src/elixir_expand.erl

```diff
@@ -528,7 +528,7 @@ expand_fn_capture(Meta, Arg, E) ->
   case elixir_fn:capture(Meta, Arg, E) of
     {{remote, Remote, Fun, Arity}, EE} ->
       is_atom(Remote) andalso
-        elixir_lexical:record_remote(Remote, ?key(E, function), ?key(E, lexical_tracker)),
+        elixir_env:trace({remote_function, Meta, Remote, Fun, Arity}, E),
```

Comment:
Should `Meta` be the `AttachedMeta` below?

Reply:
It doesn't need to because anyone using tracing can compute the attached meta using `E`. The reason we have AttachedMeta is for the code verification pass which doesn't have `E` anymore.

Reply:
Ah! Ok. I was mentioning it because we are passing `AttachedDotMeta` instead of `DotMeta` [a few lines below](https://github.com/elixir-lang/elixir/pull/9247/files#diff-ed0344c13ed25389035f337d94902894R804).

Reply:
Yeah, I would rather not pass it. Good catch!